include/libremidi/backends/winmm/midi_in.hpp

```diff
@@ -74,12 +74,13 @@ class midi_in_winmm final
     }
 
     // Allocate and init the sysex buffers.
-    this->sysexBuffer.resize(configuration.sysex_buffer_count);
-    for (int i = 0; i < configuration.sysex_buffer_count; ++i)
+    const auto bufferCount = static_cast<std::size_t>(configuration.sysex_buffer_count);
+    this->sysexBuffer.resize(bufferCount);
+    for (std::size_t i = 0; i < bufferCount; ++i)
     {
-      this->sysexBuffer[i] = (MIDIHDR*)new char[sizeof(MIDIHDR)];
+      this->sysexBuffer[i] = reinterpret_cast<MIDIHDR*>(new char[sizeof(MIDIHDR)]);
```

Comment:
this can definitely be a static_cast (in general it's better as it has a few% more type-safety and intelligence built-in than reinterpret_cast which will just do the dumb thing)

Reply:
Yeah, these ones were just automated conversions my IDE was doing and I hadn't checked if they could be static cast or not. Actually now that I think about it, couldn't this be replaced with this?
```cpp
this->sysexBuffer[i] = new MIDIHDR;
```

Reply:
This seems to work, so I just went with this.